ramls/titles.raml

```diff
@@ -44,3 +44,27 @@ resourceTypes:
       collection-item:
         exampleItem: !include acq-models/mod-orders-storage/examples/title_get.sample
         schema: title
+
+  /unlink/{id}:
+    uriParameters:
+      id:
+        description: The UUID of a Title
+        type: UUID
+    delete:
```

Comment:
The ticket description mentions a POST method. Description should be updated if DELETE is used instead. I feel like POST makes more sense than DELETE, as we are not deleting the title or an "unlink" resource. I think DELETE should only be used to delete a resource by id.

Reply:
POST is more suitable for Unlinking operations for me as well, DELETE is for deleting title

Reply:
After discussion, we agreed to move DELETE orders/titles/{id} endpoint